tests/asserts/final/publication.py

```diff
@@ -0,0 +1,17 @@
+from layman import app, util as layman_util
+
+
+def info_items(workspace, publ_type, name):
```

Comment:
In one of following commits, I've realized it would be nice to use really atomic methods with descriptive names.

I tried to understand and describe, what we are testing here. I see two possible methods:
- source_has_its_key_or_it_is_empty
- source_internal_keys_are_subset_of_source_sibling_keys